test/language/identifiers/other_id_continue-astral.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-names-and-keywords
+description: Test grandfathered astral characters of ID_Continue.
```

Comment:
“grandfathered” is the opposite of what happened here: astral characters were not valid in identifiers prior to ES2015. (Same in the other file.)